prepare-vms/scripts/postprep.rc

```diff
@@ -31,22 +31,29 @@ STEP = 0
 START = time.time()
 
 def bold(msg):
-    return "{} {} {}".format("$(tput smso)", msg, "$(tput rmso)")
+    return "{} {} {}".format("", msg, "")
```

Comment:
Do you remember why you removed the `tput` invocations from here?

Reply:
This looks like a mistake to me. The issue I was probably trying to resolve should be handled by the removal of lines 42-43.